tests/migration_test.go

```diff
@@ -1001,6 +1001,29 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 			})
 
+			It("[test_id:6970]should migrate vmi with a usb disk", func() {
```

Comment:
We have the `test_id:1853` test that we normally run to test upgrades. I think it might be a good idea to add the USB disk in that test as well.

Reply:
Done, thank you.